### PR TITLE
[api-migration-hackathon] Contour around point

### PR DESCRIPTION
**Pull request by kappe-c**
Thanks for contributing to TTK!

Before submitting your pull request, please:

- [x] Review our [Contributor Guidelines](https://github.com/topology-tool-kit/ttk/blob/dev/CONTRIBUTING.md), in particular regarding code formatting (with clang-format) and continuous integration.

- [x] Please provide a quick description of your contributions below:

In context of ttkSummerHackathon2020
(also including some new stuff since the last pull request, for a new paper to be published)

### CODE REVIEW

**Comment by julien-tierny** [comment]
hi christopher, thanks a lot for the PR!

there seems to be some build issues under windows.
you may want to double check headers and linking variables (vtk.module and ttk.module)

File: core/vtk/ttkContourAroundPoint/ttkContourAroundPoint.cpp [hunk 1]
```diff
@@ -19,116 +22,132 @@
...
+  if(!triangulation)
     return false;
-  }
 
-  ttk::Triangulation *triangulation
-    = ttkTriangulation::getTriangulation(dataset);
-  if(!triangulation) {
-    vtkErrorMacro("No ttk::Triangulation could be gained from the input "
-                  "field");
+  auto scalars = GetInputArrayToProcess(0, dataset);
```
Comment (julien-tierny) [inline core/vtk/ttkContourAroundPoint/ttkContourAroundPoint.cpp:92]: can you please PR ttk-data to update your state file?
see https://github.com/topology-tool-kit/ttk-data/commit/ba0daada042a2c78dddb0c8700db9639d2791162 for an example.

File: standalone/ContourAroundPoint/main.cpp [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+#include <vtkXMLGenericDataObjectReader.h>
+
+int main(int argc, char **argv) {
+
+  // ---------------------------------------------------------------------------
+  // Program variables
+  // ---------------------------------------------------------------------------
+  std::vector<std::string> inputFilePaths;
+  std::vector<std::string> inputArrayNames;
+  std::string outputArrayName{"AveragedArray"};
```
Comment (julien-tierny) [inline standalone/ContourAroundPoint/main.cpp]: that line you can remove (it was just for the example)

**Comment by julien-tierny** [comment]
overall, it looks good.
thanks for addressing my few minor remarks.

you'll need to PR ttk-data to port your example before I can merge that PR.
thanks!

**Comment by kappe-c** [comment]
... and the data repo companion PR is made now

**Comment by julien-tierny** [comment]
alright, looks good.
thanks!

**Pull request merged**